### PR TITLE
Hide PKI Client database password in log file

### PR DESCRIPTION
**Pull request by Akasurde**
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

**Comment by stlaz** [comment]
You will need to do something similar in `ipaserver/install/krainstance.py` as well.

File: ipaserver/install/krainstance.py [hunk 1]
```diff
@@ -150,6 +150,7 @@ def __spawn_instance(self):
         os.chown(cfg_file, pent.pw_uid, pent.pw_gid)
         self.tmp_agent_db = tempfile.mkdtemp(
                 prefix="tmp-", dir=paths.VAR_LIB_IPA)
+        self.tmp_agent_pwd = ipautil.ipa_generate_password()
```
Comment (stlaz) [inline ipaserver/install/krainstance.py]: We are not using the password outside the scope of this method so please reduce this scope as well (remove `self.`)

Comment (stlaz) [inline ipaserver/install/krainstance.py]: We are not using the password outside the scope of this method so please reduce this scope as well (remove `self.`)

**Comment by stlaz** [comment]
Works well, thanks!

**Comment by MartinBasti** [comment]
master:

* 7fddc1df573cb56949b1bc8ad83a041e97523df1 Hide PKI Client database password in log file

**Comment by Akasurde** [comment]
@stlaz @MartinBasti Thanks.

**Pull request closed without merging**